templates/packages/order_edit.html

```diff
@@ -0,0 +1,254 @@
+{%extends 'base.html' %}
+
+{% block title%}
+     הזמנה {{order.collection_date.date}}
+{%endblock%}
+
+{% block content %}
+    <div class="row justify-content-center mb-3">
+        <div class="col-auto order-2">
+            <a class="btn btn-outline-dark" data-bs-toggle="modal" data-bs-target="#package-modal" data-bs-origin-street="{{partner.address.street_name}}" data-bs-origin-street-number="{{partner.address.street_number}}" data-bs-origin-city="{{partner.address.city.name}}" data-bs-origin-area="{{partner.address.city.area}}">חבילה חדשה +</a>
+        </div>
+        {%if is_staff%}
+            <div class="col-auto order-1">
+                <form method="post"> {% csrf_token %}

```

Comment:
You might find it easier to use `action=` to direct into into different views, instead of the same view with `update-type`.

Reply:
Opened an [issue](https://github.com/dandeduck/package-tracking-web/issues/21)

Reply:
#21